app/controllers/static_pages_controller.rb

```diff
@@ -4,7 +4,7 @@ def index
   end
      
   def home
-    
+    @order_items = OrderItem.new
```

Comment:
@order_items is not an array, why plural?

Reply:
@PopaFlorian  check this please